packages/editor/src/store/constants.ts

```diff
@@ -21,6 +21,7 @@ export const TEMPLATE_POST_TYPE = 'wp_template';
 export const TEMPLATE_PART_POST_TYPE = 'wp_template_part';
 export const PATTERN_POST_TYPE = 'wp_block';
 export const NAVIGATION_POST_TYPE = 'wp_navigation';
+export const PAGE_POST_TYPE = 'page';
```

Comment:
It seems we don't use this anywhere.

Reply:
`PAGE_POST_TYPE ` is used in setAsHomepage on [line 204](https://github.com/WordPress/gutenberg/pull/65426/files#diff-3488c5c4b35c5d05f5d70c88647016bd7e62a7b64b08bbf773099abd642edf9eR204).

Reply:
It's declared [there](https://github.com/WordPress/gutenberg/pull/65426/files#diff-3488c5c4b35c5d05f5d70c88647016bd7e62a7b64b08bbf773099abd642edf9eR23). So probably you wanted to import this at that file.

Reply:
Oh I see - it looks like the intention was to import this from there but that's in a different package (in editor rather than fields), maybe in that case it's best to just declare it at the top of the file and remove the constant.

Reply:
I ended up removing this constant so it doesn't need to be imported: https://github.com/WordPress/gutenberg/pull/65426/commits/81b76f86df3516535620c2080996a6a38847e4a1.